.github/workflows/ci.yml

```diff
@@ -43,6 +43,7 @@ jobs:
           fail_ci_if_error: true
 
   test_napari:
+    if: ${{ github.event.label.name != 'test-all-plugins' }}
```

Comment:
It looks like this triggers for prs with the test-all-plugins label.  Is that right?